pkgs/by-name/ve/versionInstallHook/hook.sh

```diff
@@ -0,0 +1,56 @@
+_handleCmdOutput(){
+    local versionOutput cmdProgram chdir
+    cmdProgram=./"$(basename "$1")"
+    chdir="$(dirname "$1")"
```

Comment:
I'd prefer if a different workdir is used, since this will pass on programs which only works when in CWD, a pretty serious failure mode.

Reply:
> I'd prefer if a different workdir is used, since this will pass on programs which only works when in CWD, a pretty serious failure mode. Root might be a decent alternative

Yes so this was meant to handle a certain different edge case which I now take care of differently. I used `--chdir=/` which is a good idea.